ksql-engine/src/main/java/io/confluent/ksql/function/FunctionRegistry.java

```diff
@@ -151,6 +153,27 @@ private void init() {
         "EXTRACTJSONFIELD", JsonExtractStringKudf.class);
     addFunction(getStringFromJson);
 
+    KsqlFunction jsonArrayContainsString = new KsqlFunction(
+            Schema.BOOLEAN_SCHEMA, Arrays.asList(Schema.STRING_SCHEMA, Schema.STRING_SCHEMA),
+            "JSON_ARRAY_CONTAINS", JsonArrayContainsKudf.class);
```

Comment:
I have doubts should name contains underscore or not? For consistency with `EXTRACTJSONFIELD` it should be without, but for readability, I argue that underscore is needed.

Reply:
@satybald  - it needs to be consistent for this PR. EXTRACTJSONFIELD - changing formatting will require doc changes etc - which could be done in another PR

Reply:
@bluemonk3y good point about the doc. I completely forgot to document function in the doc :smile: